File: Assignments/Solutions/Shreya Kundu/Assignment 5/2.cpp
/*
Question 2: Given an array arr[] of N positive integers and a number K. The task is to find the kth smallest element in the array.
Input: 
N = 5, K = 3
arr[] = {3,5,4,2,9}
Output: 
4
*/
#include<bits/stdc++.h>
using namespace std;
int main(){
	int n,k;
	cin>>n>>k;
	int a[n];
	for(int i=0;i<n;i++)
	   cin>>a[i];
    for(int i=0;i<n-1;i++)
       for(int j=i+1;j<n;j++)
         if(a[j]<a[i])
           swap(a[i],a[j]);
    cout<<a[k-1];
	return 0;
}

File: Assignments/Solutions/Utkarsh Rastogi/assignment3/2.py
'''Question2:
Given a 2D array arr[][] of size M x N integers where M is the number of rows and N is the number of columns. The task is to reverse every row of the given 2D array.
Example: 
 

Input: arr[][] = 
{ {1, 2, 3},
  {4, 5, 6},
  {7, 8, 9} }
Output:
3 2 1
6 5 4
9 8 7'''

matrix=[]
rows=int(input('enter the number of rows-:'))
cols=int(input('enter the number of columns-:'))

for i in range(rows):
    a=[]
    for j in range(cols):
        a.append(int(input(f'Enter the element {i}{j}-:')))
    matrix.append(a) 
print(matrix)

for i in range(rows):
    
    for j in range(cols):
        print(f'|  {matrix[i][j]}  |', end='  ')  
    print('\n')  

print('The reverse row matrix is-:')

#reversing rows
    
for i in range(rows):
    
    for j in range(cols-1,-1,-1):
        print(f'|  {matrix[i][j]}  |', end='  ')  
    print('\n')  
